src/hotspot/os/aix/os_aix.cpp

```diff
@@ -1162,7 +1162,7 @@ static void* dll_load_library(const char *filename, char *ebuf, int ebuflen) {
 /*
  Load library named <filename>
  Search order:
- filename-> load "libfilename.so" first,then load libfilename.a,on failure. 
+ libfilename-> load "libfilename.so" first,then load libfilename.a,on failure. 
  In,OpenJ9,the libary with .so extension is loaded first and then .a extension,on failure.
```

Comment:
Hi Suchi, I'm puzzled. Your comment implies for me, that load library gets a 'base' filename without 'lib' prefix and without extension (e.g. 'name'). Then the j9 code creates the filename 'libname.so' first and on failure 'libname.a' second. What about given libname.so explicitly (e.g. libname.so)? Does j9 really use 'libname.a' as a failure fallback in this case?

Reply:
The load library gets the entire library name, after construction from dll_build_name. This is always a .so file name. When .so file name fails to load, we fallback to .a filename. 
Do i need to mention the filename as libfilename.so then ?

Reply:
Yes, I think this would make it clear.

Reply:
Please use space after comma.